docroot/modules/custom/mass_validation/mass_validation.module

```diff
@@ -1191,6 +1191,9 @@ function mass_validation_form_node_org_page_form_alter(&$form, FormStateInterfac
  *   The form array passed by reference.
  */
 function mass_validation_org_page_alter_helper(array &$form) {
+  if (isset($form['field_boards'])) {
```

Comment:
Would it be better to hide the field in the form display config since the field will eventually be removed?

Reply:
Yeah makes sense to me. I just decided to not deal with the configs for now, for the same reason, since we will be migrating it later. I can change this to work from the Form display, but we can keep this for now, since we will revisit it later during data migration work. @weekbeforenext